data/json/items/armor.json

```diff
@@ -10997,7 +10997,7 @@
         "environmental_protection" : 3,
         "encumbrance" : 40,
         "bashing" : 10,
-        "flags" : ["OVERSIZE", "VARSIZE", "WATERPROOF", "STURDY", "OUTER"],
+        "flags" : ["OVERSIZE", "VARSIZE", "WATERPROOF", "STURDY", "OUTER", "ALLOWS_NATURAL_ATTACKS"],
```

Comment:
I assume head-based natural attacks are antlers or bullhorns, those would at the least tear the hood if used, better to assume they interfere.